src/components/dashboard/__tests__/CollapsibleDashboardFilters.test.tsx

```
@@ -0,0 +1,413 @@
+      expect(screen.getByRole('button')).toBeInTheDocument();
+      
+      consoleSpy.mockRestore();
+    });
+  });
+
+  describe('server-side rendering', () => {
+    beforeEach(() => {
+      // Mock window as undefined to simulate SSR
+      const originalWindow = global.window;
+      // @ts-ignore
+      delete global.window;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)